zebra-chain/src/block/tests.rs

```diff
@@ -1,5 +1,11 @@
 // XXX generate should be rewritten as strategies
+#[cfg(feature = "bench")]
+pub mod generate;
+#[cfg(test)]
 mod generate;
```

Comment:
The second `mod generate` is redundant and causes a clippy warning:
```suggestion
#[cfg(any(test, feature = "bench"))]
pub mod generate;
```